Packages/Components/Sources/StateView/StateEmptyView.swift

```diff
@@ -5,17 +5,97 @@ import SwiftUI
 import Style
 
 public struct StateEmptyView: View {
+    var title: TextValue
+    let description: TextValue?
 
-    let message: String
-    
-    public init(message: String) {
-        self.message = message
+    let image: Image?
+
+    public init(
+        title: String,
+        titleTextStyle: TextStyle = .headline,
+        description: String? = nil,
+        descriptionTextStyle: TextStyle = .footnote,
+        image: Image? = nil
+    ) {
+        // set regular font if we have only title
+        let titleStyle = (image == nil && description == nil) ? TextStyle(font: .body, color: titleTextStyle.color) : titleTextStyle
+        let titleValue = TextValue(text: title, style: titleStyle)
+        let descriptionValue = description.map { TextValue(text: $0, style: descriptionTextStyle) }
+
+        self.init(titleValue: titleValue, descriptionValue: descriptionValue, image: image)
+    }
+
+    public init(
+        titleValue: TextValue,
+        descriptionValue: TextValue? = nil,
+        image: Image? = nil
+    ) {
+        self.title = titleValue
+        self.description = descriptionValue
+        self.image = image
     }
-    
-    public var body: some View {
-        VStack {
-            Text(message)
-        }
-        .tint(Colors.black)
+
+     public var body: some View {
+         VStack(spacing: Spacing.medium) {
+             image?
+                 .resizable()
+                 .aspectRatio(contentMode: .fit)
+                 .frame(width: Sizing.image.chain, height: Sizing.image.chain, alignment: .center)
+                 .foregroundStyle(Colors.gray)
+
+             VStack(spacing: description == nil ? 0 : Spacing.tiny) {
+                 Text(title.text)
+                     .textStyle(title.style)
+                     .multilineTextAlignment(.center)
+
+                 if let description = description {
```

Comment:
```suggestion
                 if let description {
```

use this style, much shorter